ui/transactions_page.go

```diff
@@ -58,11 +58,11 @@ func (win *Window) TransactionsPage(common pageCommon) layout.Widget {
 		filterSortW:            new(widget.Enum),
 		defaultFilterSorter:    "0",
 		defaultFilterDirection: "0",
-		rowDirectionWidth:      .04,
-		rowDateWidth:           .2,
-		rowStatusWidth:         .2,
-		rowAmountWidth:         .3,
-		rowFeeWidth:            .26,
+		txnColumn1Width:        .04,
+		txnColumn2Width:        .2,
+		txnColumn3Width:        .2,
+		txnColumn4Width:        .3,
+		txnColumn5Width:        .26,
```

Comment:
You should use the names of the columns, rather than 1,2,3.  "directionColumnWidth" is just fine.